### PR TITLE
[ENG-459] feat: Add hubspot write tests

### PR DESCRIPTION
**Pull request by eberle1080**
I forgot to add these tests to the write PR. These tests actually validate that hubspot writes work.

See https://github.com/amp-labs/connectors/pull/41 for an example run.

### CODE REVIEW

File: test/hubspot/main.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+		utils.Fail("CLIENT_ID not found in creds")
+	}
+
+	clientSecret, found := config["CLIENT_SECRET"]
+
+	if !found {
+		utils.Fail("CLIENT_SECRET not found in creds")
+	}
+
+	accessToken, found := config["ACCESS_TOKEN"]
```
Comment (laurenzlong) [inline test/hubspot/main.go:99]: access token is not needed from the file if it's going to be refreshed anyways.

File: test/utils/creds.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import (
+	"os"
+
+	"github.com/spyzhov/ajson"
+)
+
+// Credentials returns the credentials from the creds.json file.
+func Credentials() (*ajson.Node, error) {
+	fileName := "creds.json"
```
Comment (laurenzlong) [inline test/utils/creds.go:11]: Can this be a parameter to the function instead? There will be more than one creds file for the different tests, within the Hubspot tests, can pass in `hubspot-creds.json` as the parameter, and within the Salesforce tests, can pass in `salesforce-creds.json`, etc.

File: test/hubspot/main.go [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+type Contact struct {
+	Email     string `json:"email"`
+	Phone     string `json:"phone"`
+	Company   string `json:"company"`
+	Website   string `json:"website"`
+	Lastname  string `json:"lastname"`
+	Firstname string `json:"firstname"`
+}
+
+func main() {
```
Comment (laurenzlong) [inline test/hubspot/main.go:30]: Can you please add comments on how to run this test, and also create a sample creds file in OnePassword so others on the team can easily get up and running with this test?

File: test/hubspot/main.go [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+	if err != nil {
+		utils.Fail("error creating hubspot connector", "error", err)
+	}
+
+	return conn
+}
+
+// getOAuthInfo returns the OAuth2 config and token from the creds.json file.
+func getOAuthInfo(creds *ajson.Node) (*oauth2.Config, *oauth2.Token) {
+	config := creds.MustObject()
```
Comment (laurenzlong) [inline test/hubspot/main.go:86]: Can you put lines 86 to 109 into a helper function that can be used by all the different provider tests not just the Hubspot test?

**Review by laurenzlong — APPROVED** [review]
Looks good, I mostly have feedback on how to make this reusable for other tests, filed https://linear.app/ampersand/issue/ENG-525/align-saleforce-connector-tests-to-use-same-helper-functions-as to align Salesforce tests with these tests.

File: test/hubspot/main.go (hunk 1 above)
Comment (eberle1080) [reply test/hubspot/main.go:99]: Fair point, I'll just make it optional then

**Pull request merged**